gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -17885,28 +17956,97 @@ properties:
     type: array
     items:
       enum:
+        - Abdomen
         - Abdominal Peritoneum
+        - Adrenal gland, NOS
+        - Adnexa
+        - Anus, NOS
+        - Appendix
+        - Bladder, NOS
+        - Bone, NOS
+        - Bone marrow
+        - Brain, NOS
```

Comment:
"Brain" seems to be missing

Reply:
Changed ticket to "Brain, NOS" per @tlicht3